docs: minor updates to log outputs and code comments in HTS script

SimiHunjan opened the pull request.

SimiHunjan committed:
Update .env.sample

Remove the name variable as it will no longer be used.

SimiHunjan committed:
Update script-transfer-hbar.js

update comment to mention transaction fees

SimiHunjan committed:
Update script-hts-ft.js

Generate an admin key and use that as the admin key for the token

SimiHunjan commented:
Fixes #9

SimiHunjan committed:
Update script-hcs-topic.js

bguiz commented on .env.sample:
```diff
@@ -3,9 +3,6 @@
 # as this file is stored as plain text on disk,
 # and is therefore not secure enough.
 
-# Name
-YOUR_NAME=
-
```
This is replicated in https://github.com/hedera-dev/hello-future-world-js/pull/8

bguiz commented on hts-ft/script-hts-ft.js:
```diff
@@ -36,7 +36,9 @@ async function scriptHtsFungibleToken() {
     logger.log('Using account:', operatorIdStr);
 
-    //ToDo: Generate a new key for the token's adminKey
+    //Generate a new key for the token's adminKey
+    const adminPrivateKey = PrivateKey.generateECDSA();
+    const adminPublicKey = adminPrivateKey.publicKey;
```
Is there a need for a separate admin key for this token in a hello world sequence?
If this is something that can be moved to a tutorial instead, we can avoid the additional complexity.

SimiHunjan replied:
The reason I added the admin key is to show that this key can be completely separate from the the account that responsible for paying for transactions fees and the treasury account key. In the past, when all of these keys are set as the same key, people get confused with the signing requirements and not understanding that if the transaction fee payer account (operator account), designated treasury account and admin key are different then the transaction requires 3 signatures and not 1. 

This step is not required if the operator key is the same as the transaction fee payer key https://github.com/hedera-dev/hello-future-world-js/blob/main/hts-ft/script-hts-ft.js#L72 because on execute the operator key automatically signs all transactions under the hood here https://github.com/hedera-dev/hello-future-world-js/blob/main/hts-ft/script-hts-ft.js#L77.

Setting the admin key is not required. If we want to reduce complexity we can remove that from being set and this example can demonstrate creating an immutable fungible token which cannot update any token properties.

SimiHunjan replied:
removed the use of the admin key

bguiz commented on transfer-hbar/script-transfer-hbar.js:
```diff
@@ -115,7 +115,7 @@ async function scriptTransferHbar() {
         ?.map((obj) => Hbar.from(obj.amount, HbarUnit.Tinybar).toString(HbarUnit.Hbar));
     logger.log(
-        'The debit and credit amounts of the transfer transaction:\n',
+        'The debit, credit, and transaction fee amounts of the transfer transaction:\n',
```
FYI @theekrystallee is looking into how to get this mapped out to each account, we can display a clearer representation here.
https://swirldslabs.slack.com/archives/C076TRA61JS/p1722568520115449?thread_ts=1722567923.867689&cid=C076TRA61JS
![image](https://github.com/user-attachments/assets/7cd322d2-9f89-4285-921f-dd8f5d4fcc72)

bguiz replied:
So I think we have an answer we can work based off of:

> They greyed out is fee accounts, 800, 98 and node IDs. Which are the accounts fees go into.

So we could simply filter out `accountId < 50 || accountId  = 98 || accountId = 800 ` and print the rest.
Or print all of them, but label fee recipients.

SimiHunjan committed:
Update script-hts-ft.js

updated FT to remove assigning an admin key

bguiz commented on hts-ft/script-hts-ft.js:
```diff
@@ -58,28 +54,18 @@ async function scriptHtsFungibleToken() {
 
-    // Sign the transaction with the account key that will be paying for this transaction
-    const tokenCreateTxSigned = await tokenCreateTx.sign(adminPrivateKey);
-
-    //ToDo: Sign with the admin key
-
-    // Submit the transaction to the Hedera Testnet
-    const tokenCreateTxSubmitted = await tokenCreateTxSigned.execute(client);
-
     // Get the transaction receipt
-    const tokenCreateTxReceipt = await tokenCreateTxSubmitted.getReceipt(client);
```
Hmm... 🤔

SimiHunjan replied:
updated PR to add these steps back thanks!

SimiHunjan committed:
Merge branch 'hedera-dev:main' into update-FT-example

SimiHunjan committed:
Update script-hts-ft.js

Update example to show freezing and signing steps for the token create transaction

bguiz commented on hts-ft/script-hts-ft.js:
```diff
@@ -56,23 +54,19 @@ async function scriptHtsFungibleToken() {
         tokenCreateTxId.toString());
 
-    // Sign the transaction with the account key that will be paying for this transaction
+    //Sign the transaction with the private key of the treasury account(operator key)
```
```suggestion
    // Sign the transaction with the private key of the treasury account (operator key)
```

bguiz commented on hts-ft/script-hts-ft.js:
```diff
@@ -81,7 +75,7 @@ async function scriptHtsFungibleToken() {
     // Get the token ID
     const tokenId = tokenCreateTxReceipt.tokenId;
-    logger.log('tokenId:', tokenId.toString());
+    logger.log("tokenId:", tokenId.toString());
```
```suggestion
    logger.log('tokenId:', tokenId.toString());
```

bguiz commented on hts-ft/script-hts-ft.js:
```diff
@@ -56,23 +54,19 @@ async function scriptHtsFungibleToken() {
         // Set the freeze default value to false
         .setFreezeDefault(false)
-        // Freeze the transaction to prepare for signing
+        //Freeze the transaction and prepare for signing
```
for a subsequent PR: something to check and make consistent across the different scripts - let's have standardised comments for the transaction config and flow.
IIRC we made some effort to do this in an earlier round, so they should be mostly consistent now - but probably have drifted apart a little too.

bguiz committed:
docs: minor - code comments/ lint
